From laurent.pinchart@skynet.be Tue Nov  7 01:12:43 2006
Content-Disposition: inline
From: Laurent Pinchart <laurent.pinchart@skynet.be>
To: linux-usb-devel@lists.sourceforge.net
Subject: USB: Fixed outdated usb_get_device_descriptor() documentation
Date: Tue, 7 Nov 2006 10:16:25 +0100
Cc: Greg KH <greg@kroah.com>
Message-Id: <200611071016.25244.laurent.pinchart@skynet.be>

usb_get_device_descriptor() used to convert several descriptor fields to host
CPU's byte order. Now that it doesn't convert them anymore, update the
documentation to reflect this.

Signed-off-by: Laurent Pinchart <laurent.pinchart@skynet.be>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/message.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

--- gregkh-2.6.orig/drivers/usb/core/message.c
+++ gregkh-2.6/drivers/usb/core/message.c
@@ -828,10 +828,7 @@ char *usb_cache_string(struct usb_device
  * Context: !in_interrupt ()
  *
  * Updates the copy of the device descriptor stored in the device structure,
- * which dedicates space for this purpose.  Note that several fields are
- * converted to the host CPU's byte order:  the USB version (bcdUSB), and
- * vendors product and version fields (idVendor, idProduct, and bcdDevice).
- * That lets device drivers compare against non-byteswapped constants.
+ * which dedicates space for this purpose.
  *
  * Not exported, only for use by the core.  If drivers really want to read
  * the device descriptor directly, they can call usb_get_descriptor() with
